deployment/ansible/roles/raster-foundry.app/tasks/static-files.yml

```diff
@@ -41,7 +41,7 @@
 
 - name: Run Django collectstatic command
   django_manage: command=collectstatic app_path="{{ django_home }}"
-  environment: django_config
+  environment: "{{ envdir_config | combine(django_config) }}"
```

Comment:
When I tried to do `environ.get('RF_EMR_DEBUG').upper() == 'TRUE'` it resulted in an error while running the collectstatic command (in ansible only) because these variables were not in scope.

Here's the error you should receive without this change:

```
TASK: [raster-foundry.app | Run Django collectstatic command] ***************** 
failed: [app] => {"cmd": "python manage.py collectstatic --noinput", "failed": true, "path": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games", "state": "absent", "syspath": ["/home/vagrant/.ansible/tmp/ansible-tmp-1445889752.14-174918482846132", "/usr/lib/python2.7", "/usr/lib/python2.7/plat-x86_64-linux-gnu", "/usr/lib/python2.7/lib-tk", "/usr/lib/python2.7/lib-old", "/usr/lib/python2.7/lib-dynload", "/usr/local/lib/python2.7/dist-packages", "/usr/lib/python2.7/dist-packages"]}
msg: 
:stderr: Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 351, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 343, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 182, in fetch_command
    settings.INSTALLED_APPS
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/opt/app/rf/settings/test.py", line 3, in <module>
    from base import *  # NOQA
  File "/opt/app/rf/settings/base.py", line 287, in <module>
    AWS_BUCKET_NAME = get_env_setting('RF_S3_BUCKET')
  File "/opt/app/rf/settings/base.py", line 26, in get_env_setting
    raise ImproperlyConfigured(error_msg)
django.core.exceptions.ImproperlyConfigured: Set the RF_S3_BUCKET env variable
```